docs/Introduction/whatIsLitProtocol.md

```diff
@@ -47,7 +47,7 @@ Lit’s access control protocol gives individuals the ability to **read** privat
 
 To facilitate this second use case we created two things: Lit Actions and Programmable Key Pairs (PKPs). Lit Actions are immutable JavaScript functions stored on IPFS. Actions can be thought of as smart contracts with superpowers: they have network access and the ability to make arbitrary HTTP requests. 
 
-PKPs are the public/private key pairs generated by the Lit network and are minted in the form of an ERC-721 NFT. The owner of the NFT becomes the sole controller of the underlying private key. 
+PKPs are the public/private key pairs generated by the Lit network and are minted in the form of an ERC-721 NFT. Only the owner of the NFT can combine the underlying key shares and form the complete signature. 
```

Comment:
A correction for this is, "Only those with authorized access to the NFT can combine the underlying key shares..."

Since we now have multiple ways to authenticate and use PKPs!